tests/helpers.py

```diff
@@ -294,6 +304,24 @@ def create_pvc(sc_name, pvc_name=None, size=None, wait=True):
     return ocs_obj
 
 
+def create_multiple_pvc(sc_name, number_of_pvc=1, size=None, namespace=defaults.ROOK_CLUSTER_NAMESPACE):
```

Comment:
@ebenahar since PVCs have "accessModes", will it help to have this included in param as None ? If required, user can send a PVC create request for a particular accessModes ?
But catch is to use these in POds as they behave differently.

Reply:
@ebenahar why is the namespace = defaults.ROOK_CLUSTER_NAMESPACE ? Shouldn't it be equal to the new  test projects we create and pass as argument here ?

Reply:
Just a caution here. We should not end up in passing too many parameters....

Reply:
@nehaberry regarding the namespace, I set defaults.ROOK_CLUSTER_NAMESPACE as the default one in order to not break other places that call this function and rely on this default namespace.
I'll try to change it so it'll be mandatory to pass a namespace

Reply:
@nehaberry regarding accessModes - I suggest we add this here once it'll be needed